src/octoprint/printer/standard.py

```diff
@@ -927,6 +927,8 @@ def is_ready(self, *args, **kwargs):
         return (
             self.is_operational()
             and not self.is_printing()
+            and not self._comm.isBusy()
```

Comment:
If you leave `is_printing` in there this is redundant. Just `self._comm.isBusy` covers all printing and paused states.

Reply:
OK, removed, and fixed the merge conflict, I was using a stale fork.